src/handlers.ts

```diff
@@ -450,3 +483,62 @@ export async function inboxHandler(
         },
     );
 }
+
+export async function getActivities(
+    ctx: Context<{ Variables: HonoContextVariables }>,
+) {
+    const DEFAULT_LIMIT = 10;
+
+    const db = ctx.get('db');
+    const globaldb = ctx.get('globaldb');
+    const apCtx = fedify.createContext(ctx.req.raw as Request, {db, globaldb});
+
+    // Parse cursor and limit from query parameters
+    const cursor = parseInt(ctx.req.query('cursor') || '0', 10);
+    const limit = parseInt(ctx.req.query('limit') || DEFAULT_LIMIT.toString(), 10);
+
+    // Fetch the liked items from the database:
+    //   - Data is structured as an array of strings
+    //   - Each string is a URI to an object in the database
+    // This is used to add a "liked" property to the item if the user has liked it
+    const liked = (await db.get<string[]>(['liked'])) || [];
+
+    // Fetch the inbox from the database:
+    //   - Data is structured as an array of strings
+    //   - Each string is a URI to an object in the database
+    const inbox = (await db.get<string[]>(['inbox'])) || [];
+
+    // Slice the results array based on the cursor and limit
+    const paginatedInbox = inbox.slice(cursor, cursor + limit);
```

Comment:
I think we want to slice from the end of the array, so that we get the most recent items first

Reply:
Yeh, previously we was sorting in the client. I've updated this now so the newest items are returned first and removed the manual sorting in the client 👍